tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__;
+        });
+    }
+
+    private function getMix()
+    {
+        return new \Illuminate\View\Mix\Mix;
```

Comment:
I believe there is some misunderstanding out there.
You plan to refactor a mix function that everyone is using on `5.4` impacting current users, and that will be soon merged to `5.5`.
This is a function loaded on each page request displaying a view, so it's expected to be performant and highly tested, right?

No need to be pissed and get personal